Dockerfile.development

```diff
@@ -8,12 +8,15 @@ ENV NODE_ENV=development
 RUN apk update && apk upgrade && \
     # Build dependencies for node_modules
     apk add --virtual native-deps \
-    g++ gcc libgcc libstdc++ linux-headers autoconf automake make nasm python git curl \
+    # Python version must be specified starting in alpine3.12
+    g++ gcc libgcc libstdc++ linux-headers autoconf automake make nasm python2 git curl \
```

Comment:
will python3 not work? python2 has reached end of life

Reply:
as above, changed to python3